Tests/SentryTests/SentryClientTests.swift

```diff
@@ -169,6 +169,27 @@ class SentryClientTest: XCTestCase {
         }
     }
     
+    func testCaptureEventWithScope_SerializedTagsAndExtraShouldMatch() {
```

Comment:
I wonder why did you add this test? This should already be tested somewhere.

Reply:
If this was tested before,  the [1311](https://github.com/getsentry/sentry-cocoa/issues/1311) issue won't happen :) 

This is somehow an integration test. It considers Event, Transaction, Scope. The logic for this thing is inside this method.

`- (SentryId *)sendEvent:(SentryEvent *)event
                  withScope:(SentryScope *)scope
     alwaysAttachStacktrace:(BOOL)alwaysAttachStacktrace
               isCrashEvent:(BOOL)isCrashEvent
    additionalEnvelopeItems:(NSArray<SentryEnvelopeItem *> *)additionalEnvelopeItems`
     
   `[self getTraceStateWithEvent:event withScope:scope]` - this part.
   
   Maybe I'm getting it wrong, but this is SentryClient's logic.